spi/build.gradle

```diff
@@ -46,7 +47,8 @@ dependencies {
     compileOnly "org.opensearch:opensearch:${opensearch_version}"
 
     testImplementation "org.opensearch.test:framework:${opensearch_version}"
-    testImplementation "org.apache.logging.log4j:log4j-core:${versions.log4j}"
+    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
```

Comment:
may be we can add the same mockito version we have in the build.gradle file of ml-algorithms?